recipes/tests.rb

```diff
@@ -189,7 +189,17 @@
 ###################
 # Amazon Time Sync
 ###################
-execute 'check chrony' do
+if node['init_package'] == 'init'
```

Comment:
What about using the [service resource](the https://docs.chef.io/resource_service.html)?

Reply:
could not be used for that purpose